rskj-core/src/main/java/org/ethereum/vm/trace/Op.java

```diff
@@ -37,8 +37,8 @@
     private long gas;
     private long gasCost;
 
-    private List<String> memory = new ArrayList<>();
-    private List<String> stack = new ArrayList<>();
+    private List<String> memory = new ArrayList<>(); // lgtm [java/unused-container]
```

Comment:
why do we need these two fields, if we don't ever read them?

Reply:
Could you add a comment describing how these variables are used? (JSON serialization)

Reply:
Done!